lib/chef/platform/service_helpers.rb

```diff
@@ -17,108 +17,37 @@
 #
 
 require_relative "../chef_class"
-require_relative "../mixin/train_helpers"
+require "chef-utils" if defined?(ChefUtils::CANARY)
 
 class Chef
   class Platform
+    # FIXME: deprecated, use ChefUtils::Service instead
```

Comment:
We should stick a @deprecated on all the classes we're making obsolete with this so they get marked in yard docs